profiles/permissions.easy

```diff
@@ -58,9 +58,12 @@
 /sbin/unix2_chkpwd                                      root:shadow       4755
 
 # squid changes from bnc#891268
+# - adjusted paths in bsc#1171569
 /var/cache/squid/                                       squid:root        0750
 /var/log/squid/                                         squid:root        0750
-/usr/sbin/pinger                                        root:squid        0750
+/usr/lib/squid/pinger                                   root:squid        0750
+ +capabilities cap_net_raw=ep
+/usr/lib64/squid/pinger                                 root:squid        0750
```

Comment:
While we're fixing the file system location here, shouldn't this be in libexec?

Reply:
Probably it belongs in libexec. Doing little steps here, the executable was in /usr/sbin previously. I still didn't get an answer in bsc#1171164 how this migration is supposed to continue. I don't feel particularly responsible for fixing all the remaining packages.

Reply:
I'm not proposing that we migrate all of Factory. Just that while this file is being moved the correct location, it should be moved to the now more-correct place.

I just said so in the corresponding bug.

Reply:
Since my previous sr# for squid was faulty I am currently preparing a new one that fixes the original issue and also changes the paths to libexec. I don't know if the maintainer will be happy with this but let's try. Meanwhile I added and pushed compatibility entries for all locations in lib*/squid so that we can progress with this PR#.

Reply:
OK, great!